functions/transformations/difference_test.go

```diff
@@ -223,33 +223,36 @@ func TestDifference_Process(t *testing.T) {
 				},
 			}},
 		},
-		{
-			name: "float with tags",
-			spec: &transformations.DifferenceProcedureSpec{
-				Columns: []string{execute.DefaultValueColLabel},
+		// TODO(affo): won't pass until https://github.com/apache/arrow/issues/3270 get's fixed
```

Comment:
I missed this on my first pass, but it looks like this is the only transformation that slices arrow arrays. I think we should wait for https://github.com/apache/arrow/pull/3275 and https://github.com/apache/arrow/pull/3283 to land so we don't introduce breaking changes.

Reply:
There is also limit actually #570 

But its tests passes because string columns have always the same value I think.

Anyhow that's ok.